Scripts/Proxies/TwoHandedProxyBase.cs

```diff
@@ -161,6 +182,100 @@ public virtual void Update()
 				m_RightHand.localRotation = trackedObjectInput.rightRotation.quaternion;
 			}
 		}
+
+		public void AddFeedbackRequest(FeedbackRequest request)
+		{
+			var proxyRequest = request as ProxyFeedbackRequest;
+			if (proxyRequest != null)
+			{
+				m_FeedbackRequests.Add(proxyRequest);
+				ExecuteFeedback(proxyRequest);
+			}
+		}
+
+		void ExecuteFeedback(ProxyFeedbackRequest changedRequest)
```

Comment:
Function seems to be utilized in a weird way?  When called through AddFeedbackRequest, it should always match matchChanged and sameCaller and thus continue on.

When called through RemoveFeedbackRequest, it will always fail the matchChanged and sameCaller request and essentially do nothing.

Reply:
It won't always fail matchChanged. We match on everything but the caller. This happens specifically because of the case where you remove feedback that was overriding an existing piece of feedback. This happens in the case of vacuum tool overriding the "crawl" feedback on the grip and then removing itself. When you remove the vacuum feedback, the crawl feedback matches and executes.

I ended up fixing a bug with this logic during my second Feedback pass, if that version makes more sense... https://github.com/Unity-Technologies/EditorVR/commit/4cae9b52ca01ba6bcf9cc4d8d87352d69ad57c97#diff-772d21b9cb82f8b9c5868499ff7446e9R220